packages/cdk8s-cli/test/__snapshots__/integration.test.js.snap

```diff
@@ -87,9 +87,9 @@ Object {
     },
   },
   "description": "dummy",
-  "fingerprint": "HD4O8JPi7/yzcp+ASh//v9CjPwF8WUUjrfUevSbCO9A=",
+  "fingerprint": "mdQ/Dby2xkDPeGSW31K8SNn/rUvqDSFgyzbNqu5N96c=",
   "homepage": "http://repo",
-  "jsiiVersion": "1.0.0 (build eb2db9d)",
+  "jsiiVersion": "1.1.0 (build df55f5e)",
```

Comment:
Please revert to 0.22.0 (the AWS CDK still uses this version due to some issue we are having).

Reply:
Done!